### PR TITLE
[FEAT] Initial node state simulation

### PR DESCRIPTION
**Pull request by sindhu-106**
Signed-off-by: sindhuja <sindhuja.gajula@intel.com>

### CODE REVIEW

File: config-files/InitialNodeState.json [hunk 1]
```diff
@@ -0,0 +1,851 @@
+{ "Components": [
```
Comment (paul-amonson) [inline config-files/InitialNodeState.json:1]: Doesn't this belong in the docker-deploy/eventsim-config folder? BTW: docker-deploy is going away with the new RPM installers.

File: config-files/NodeState.json [hunk 2]
```diff
@@ -0,0 +1,851 @@
+{ "Components": [
```
Comment (paul-amonson) [inline config-files/NodeState.json:1]: Same comment as previous file.

File: docker-deploy/eventsim-config/EventSim.json [hunk 3]
```diff
@@ -8,7 +8,8 @@
       "hw-inv-discover-status-url" : "/Inventory/DiscoveryStatus",
       "sensor-metadata": "/resources/ForeignSensorMetaData.json",
       "ras-metadata": "/resources/ForeignEventMetaData.json",
-      "jobs-metadata" : "/resources/ForeignJobsMetaData.json"
+      "jobs-metadata" : "/resources/ForeignJobsMetaData.json",
+      "node-state" : "/opt/ucs/etc/NodeState.json"
```
Comment (paul-amonson) [inline docker-deploy/eventsim-config/EventSim.json:12]: You aren't referencing the InitialNodeState.json here???

**Review by paul-amonson — CHANGES_REQUESTED** [review]
It appears you need some cleanup. If I am wrong just ping me.

File: config-files/InitialNodeState.json (hunk 1 above)
Comment (sindhu-106) [reply config-files/InitialNodeState.json]: I added in docker-deploy too

File: config-files/NodeState.json (hunk 2 above)
Comment (sindhu-106) [reply config-files/NodeState.json:1]: I added in docker-deploy too. Yes Paul I need to clean up one of the NodeState.json

**Review by paul-amonson — DISMISSED** [review]

**Review by yhuan47 — DISMISSED** [review]

File: eventsim/src/main/java/com/intel/dai/eventsim/NodeState.java [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+ * fetch node states.
+ */
+public class NodeState {
+
+    /**
+     * This method used to fetch node states data.
+     * @return node states data
+     * @throws SimulatorException when unable to find configuration file or process data.
+     */
+    PropertyDocument getNodeSates() throws SimulatorException {
```
Comment (nbwhite) [inline eventsim/src/main/java/com/intel/dai/eventsim/NodeState.java:24]: Typo in this method name

**Review by nbwhite — COMMENTED** [review]
One small change needed

File: eventsim/src/main/java/com/intel/dai/eventsim/NodeState.java (hunk 4 above)
Comment (paul-amonson) [reply eventsim/src/main/java/com/intel/dai/eventsim/NodeState.java:24]: Nice catch! :)

Comment (sindhu-106) [reply eventsim/src/main/java/com/intel/dai/eventsim/NodeState.java]: Good catch Nathan. Fixed it :)

**Review by nbwhite — APPROVED** [review]

**Review by paul-amonson — APPROVED** [review]

**Pull request merged**